modules/search/instant-search/README.md

```diff
@@ -31,22 +31,10 @@ This also works on WP.com
 
 ### Testing Instructions
 
-1. Add `define( "JETPACK_SEARCH_PROTOTYPE", true );` to your `wp-config.php`. If using Jetpack's Docker development environment, you can create a file at `/docker/mu-plugins/instant-search.php` and add the define there.
+1. Ensure that your site has a Jetpack Search plan. You can enable the instant search experience in the Performance tab within the Jetpack menu (/wp-admin/admin.php?page=jetpack#/performance).
 
-2. Ensure that your site has the Jetpack Pro plan and Jetpack Search enabled. You can enable Jetpack Search in the Performance tab within the Jetpack menu (/wp-admin/admin.php?page=jetpack#/performance).
+2. Select a theme of your choice and add a Jetpack Search widget to the site via the Customizer. If using a theme with a sidebar widget area, please add the Jetpack Search widget there.
 
-3. Select a theme of your choice and add a Jetpack Search widget to the site via the customizer. If using a theme with a sidebar widget area, please add the Jetpack Search widget there.
+3. Add a Jetpack Search Widget to the Jetpack Search Sidebar and configure the filters you'd like to show in the search overlay.
 
-4. Add a Jetpack Search Widget to the Jetpack Search Sidebar and configure the filters you'd like to show in the search overlay.
-
-5. Navigate to your site's search page (e.g. `/?s=hello`) via the widget you added in (3).
-
-## Architectural Choices
```

Comment:
The architectural choices are interesting background, but probably not essential for a README.